src/vs/workbench/browser/parts/editor/breadcrumbs.ts

```diff
@@ -75,6 +75,7 @@ export abstract class BreadcrumbsConfig<T> {
 	static readonly TitleScrollbarSizing = BreadcrumbsConfig._stub<IEditorPartOptions['titleScrollbarSizing']>('workbench.editor.titleScrollbarSizing');
 
 	static readonly FileExcludes = BreadcrumbsConfig._stub<glob.IExpression>('files.exclude');
+	static readonly Separator = BreadcrumbsConfig._stub<string>('breadcrumbs.separator');
```

Comment:
@namugach From the code I see it doesn't affect display seperator in the widget, so I think it'd be better to change setting id?
```suggestion
	static readonly CopySeparator = BreadcrumbsConfig._stub<string>('breadcrumbs.copySeparator');
```